styles.css

```diff
@@ -45,12 +45,54 @@ a:active {
   color: limegreen;
 }
 
-/* lets connect links*/
+/*   /////CONNECT LINKS/////   */
```

Comment:
Style preference, but the backslashes aren't necessary, and they could cause confusion: Only `/*` `*/` comment syntax is supported in CSS.